common/src/main/scala/io/phdata/repositories/ComplianceRepositoryImpl.scala

```diff
@@ -1,47 +1,37 @@
 package io.phdata.repositories
 
-import io.phdata.models.Compliance
+import io.phdata.models.{ComplianceQuestion}
 import doobie._
 import doobie.implicits._
 
 class ComplianceRepositoryImpl extends ComplianceRepository {
 
   implicit val han = CustomLogHandler.logHandler(this.getClass)
 
-  def insertRecord(compliance: Compliance): ConnectionIO[Long] =
-    sql"""
-       insert into compliance (phi_data, pci_data, pii_data)
+  override def create(compliance: List[ComplianceQuestion], workspaceId: Long): List[ConnectionIO[Long]] =
+    compliance.map(question => Statements.insert(question.id.get, workspaceId, question.complianceGroupId.get).withUniqueGeneratedKeys[Long]("id"))
+
+  override def findByWorkspaceId(workspaceId: Long): ConnectionIO[List[ComplianceQuestion]] =
+    Statements.findByWorkspaceId(workspaceId).to[List]
+
+  object Statements {
+
+    def insert(questionId: Long, workspaceId: Long, groupId: Long): Update0 =
+      sql"""
+       insert into compliance (workspace_id, question_id, group_id)
```

Comment:
I believe it should be new table `compliance_v2` instead of old `compliance`, also I don't think we need `group_id`